README.md

```diff
@@ -200,6 +200,54 @@ Or, to format only specified files, use
 nox -s format -- example.py pygdbmi/IoManager.py
 ```
 
+### Making a release
+
+Only maintainers of the [pygdbmi package on PyPi](https://pypi.org/project/pygdbmi/) can make a release.
+
+In the following steps, replace these strings with the correct values:
+
+- `<REMOTE>` is the name of the remote for the main pygdbmi repository (for instance, `origin`)
+- `<VERSION>` is the version number chosen in step 2.
+
+To make a release:
+
+1. Checkout the `master` branch and pull from the main repository with `git pull <REMOTE> master`
+2. Decide the version number for the new release; we follow
+   [Semantic Versioning](https://semver.org/) but prefixing the version with `0.`: given a version
+   number _0.SECOND.THIRD.FOURTH_, increment the:
+   - _SECOND_ component when you make incompatible API changes
+   - _THIRD_ component when you add functionality in a backwards compatible manner
+   - _FOURTH_ component when you make backwards compatible bug fixes
+3. Update `CHANGELOG.md` to list the chosen version number instead of `## dev`
+4. Update `__version__` in `pygdbmi/__init__.py` to the chosen version number
+5. Commit your changes, for instance using `git commit -a -m 'Bump version to <VERSION> for release'`
+6. Check that the docs look fine by serving them locally with `nox -s serve_docs`
+7. You are now ready to make a release, but first check that the CI tests on master all passed
+8. Push your changes straight to `master` with `git push <REMOTE> master`
+9. Go to the [new release page](https://github.com/cs01/pygdbmi/releases/new) and prepare the
+   release:
+   - Add a tag in the form `v<VERSION>` (for example `v0.1.2.3`)
+   - Set the title to `pygdbmi v<VERSION>` (for example `pygdbmi v0.1.2.3`)
```

Comment:
I wasn't sure what this is for so I picked this based on reading https://softwareengineering.stackexchange.com/a/440017.